patches/squeekboard-patches/0003-Check-if-dbus-handler-is-null-before-using.patch

```diff
@@ -0,0 +1,27 @@
+From ac3703e80f427e0061a4b8a779b0479129dec738 Mon Sep 17 00:00:00 2001
+From: William Wold <wm@wmww.sh>
+Date: Mon, 7 Feb 2022 18:18:44 -0500
+Subject: [PATCH] Check if dbus handler is null before using
+
+---
+ src/main.rs | 4 +++-
+ 1 file changed, 3 insertions(+), 1 deletion(-)
+
+diff --git a/src/main.rs b/src/main.rs
+index 0a089bf..35392b8 100644
+--- a/src/main.rs
++++ b/src/main.rs
+@@ -118,7 +118,9 @@ mod c {
+         };
+ 
+         if let Some(visible) = msg.dbus_visible_set {
+-            unsafe { dbus_handler_set_visible(dbus_handler, visible as u8) };
++            if dbus_handler != std::ptr::null() {
++                unsafe { dbus_handler_set_visible(dbus_handler, visible as u8) };
++            }
```

Comment:
The patch is pretty simple, and will hopefully be upstreamed shortly. If upstreaming is rejected, would then opt for adding a dbus session. I'll also keep this in mind in case we need the functionality (for example to show the keyboard even when apps don't request it)

Reply:
Upstream PR: https://gitlab.gnome.org/World/Phosh/squeekboard/-/merge_requests/530